spec/controllers/api/v1/animals_controller_spec.rb

```diff
@@ -38,14 +38,38 @@
       expect(response.status).to eq 200
       expect(response.content_type).to eq("application/json")
 
-      expect(returned_json.length).to eq 1
+      expect(returned_json.length).to eq 2
 
       expect(returned_json["animals"][0]["name"]).to eq "Shannon"
       expect(returned_json["animals"][0]["species"]).to eq "Chicken"
 
       expect(returned_json["animals"][1]["name"]).to eq "Jance"
       expect(returned_json["animals"][1]["species"]).to eq "Bird"
     end
+
+    it "should send user_role as admin if signed in user admin" do
+      user = FactoryBot.create(:user)
+      user.role = "admin"
+      user.save
+      sign_in user
+      get :index
+      returned_json = JSON.parse(response.body)
+      expect(returned_json["user_role"]).to eq "admin"
+    end
+
+    it "should send user_role as user if signed in not admin" do
+      user = FactoryBot.create(:user)
+      sign_in user
+      get :index
+      returned_json = JSON.parse(response.body)
+      expect(returned_json["user_role"]).to eq "user"
+    end
+
+    it "should send user_role as "" if user is not signed in" do
+      get :index
+      returned_json = JSON.parse(response.body)
+      expect(returned_json["user_role"]).to eq ""
+    end
```

Comment:
actually, you should be more explicit. I would have it return "guest" if the user is not signed in. that way there isnt any weird edge case errors with empty strings